### DISCUSSION TITLE
Requestusd.bond

### DISCUSSION THREAD

**Discussion by deserthoover** [RequestNetwork/requestNetwork#1437] [Questions] [answered]
Requestusd.bond is using your platform logo to market and have members be part of their team.  Is this a legitimate platform/project associated with you?

![IMG_0045](https://github.com/user-attachments/assets/882c6e7b-7ff3-4b09-8142-5058cd25ee57)

**Comment by aimen74** [comment, marked as answer]
Hi @deserthoover this project is **NOT** associated with Request Network.
